### PR TITLE
Feature: eif removal

### PR DESCRIPTION
**Pull request by lsh-0**
this is an effort to remove some of the legacy code lax has accrued in it's short life.

* it removes v1 of the api
* it removes all support for 'eif' - the article json format that existed before article-json
* it replaces the article stub creator for testing with something that goes through the business logic rather than steps around it.

### CODE REVIEW

File: src/publisher/tests/base.py [hunk 1]
```diff
@@ -39,11 +41,39 @@ def call_command(self, *args, **kwargs):
         stdout = StringIO()
         try:
             kwargs['stdout'] = stdout
-            call_command(*args, **kwargs)
+            dj_call_command(*args, **kwargs)
```
Comment (giorgiosironi) [inline src/publisher/tests/base.py]: :dark_sunglasses:

File: src/publisher/tests/base.py [hunk 2]
```diff
@@ -39,11 +41,39 @@ def call_command(self, *args, **kwargs):
...
+
+            'title': '[default]',
+            'version': 1,
+            'status': models.VOR,
+            'published': '2012-01-01T00:00:00Z'
+        }
+        struct.update(adata)        
+        delall(struct, ['journal']) # can't be serialized, not utilised anyway
+        
+        with self.settings(VALIDATE_FAILS_FORCE=False):
```
Comment (lsh-0) [inline src/publisher/tests/base.py]: finally found a use for this settings. vindicated!

**Comment by lsh-0** [comment]
I'm going to do a quick second pass on Monday and then merge this in then.

I used this tool: https://pypi.python.org/pypi/vulture  to help me hunt down some unused functions. unfortunately it's a bit noisy because of Django magic.

**Pull request merged**